tests/conf-disabled.js

```diff
@@ -0,0 +1,181 @@
+      },
+    }
+  },
+  jasmineNodeOpts: {
+    // makes default jasmine reporter not display dots for every spec
+    print: () => {}
+  },
+  beforeLaunch: function() {
+    process.on('uncaughtException', function() {
+      utils.reporter.jasmineDone();
+      utils.reporter.afterLaunch();
+    });
+
+    return new Promise(function(resolve) {
+      utils.reporter.beforeLaunch(resolve);
+    });
+  },
+  afterLaunch: function(exitCode) {
+    return new Promise(function(resolve) {
+      return request.post('http://localhost:31337/die')
+        .then(() => utils.reporter.afterLaunch(resolve.bind(this, exitCode)));
+    });
+  },
+  onPrepare: async () => {
+    jasmine.getEnv().addReporter(utils.specReporter);
+    jasmine.getEnv().addReporter(utils.reporter);
+    browser.waitForAngularEnabled(false);
+
+    // wait for startup to complete
+    await browser.driver.wait(prepServices(), 135 * 1000, 'API took too long to start up');
+
+    afterEach(() => {
+      return browser
+        .manage()
+        .logs()
+        .get('browser')
+        .then(logs => {
+          logs
+            .map(log => `[${log.level.name_}] ${log.message}\n`)
+            .forEach(log => browserLogStream.write(log));
+          browserLogStream.write('\n~~~~~~~~~~~~~~~~~~~~~\n\n');
+        });
+    });
+
+    return login(browser).then(() => runAndLog('User setup', setupUser));
+  }
+};
+
+exports.config = baseConfig;
+
+const runAndLog = (msg, func) => {
+  console.log(`API startup: ${msg}`);
+  return func();
+};
+
+const prepServices = async () => {
+  let apiReady;
+  if (constants.IS_TRAVIS) {
+    console.log('On travis, waiting for horti to first boot api');
+    // Travis' horti will be installing and then deploying api and sentinel, and those logs are
+    // getting pushed into horti.log Once horti has bootstrapped we want to restart everything so
+    // that the service processes get restarted with their logs separated and pointing to the
+    // correct logs for testing
+    await listenForApi();
+    console.log('Horti booted API, rebooting under our logging structure');
+    apiReady = await request.post('http://localhost:31337/all/restart');
+  } else {
+    // Locally we just need to start them and can do so straight away
+    apiReady = await request.post('http://localhost:31337/all/start');
+  }
+
+  await listenForApi();
+  await runAndLog('Settings setup', setupSettings);
+  await runAndLog('User contact doc setup', utils.setUserContactDocNative);
+  return apiReady;
+};
+
+const apiRetry = () => {
+  return new Promise(resolve => {
+    setTimeout(() => {
+      resolve(listenForApi());
+    }, 1000);
+  });
+};
+
+const listenForApi = async () => {
+  console.log('Checking API');
+  try {
+    const result =  await utils.requestNative({ path: '/api/info' });
+    return result;
+  } catch(err) {
+    console.log('API check failed, trying again in 1 second');
+    console.log(err.message);
```

Comment:
The only real difference here is that I'm now consloe logging out the `err.message`. Otherwise it would just show you the `API check failed, trying again in 1 second'` and you do not know why really ever. Even when it eventually timesout.